### PR TITLE
Add Pyth-over-Wormhole design document

### PR DESCRIPTION
**Pull request by drozdziak1**
This change is [Reviewable](https://reviewable.io/reviews/certusone/wormhole/781)

### CODE REVIEW

File: whitepapers/0007_pyth_over_wormhole.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+| pyth-client-rs            | Rust library for reaching Pyth on Solana                                                                                                                                                           | https://github.com/pyth-network/pyth-client-rs    |
+| Solana                    | The main codebase for Solana client libraries                                                                                                                                                      | https://github.com/solana-labs/solana             |
+| Solitaire                 | A convenient Rust framework for contract development that greatly simplifies Solana's unique execution model; A by-product of Wormhole's Solana efforts, built on top of the official Solana APIs. | https://forge.certus.one/plugins/gitiles/wormhole |
+| Wormhole's Solana program | The Solana end of Wormhole, built with Solitaire.                                                                                                                                                  | https://forge.certus.one/plugins/gitiles/wormhole |
```
Comment (SEJeff) [inline whitepapers/0007_pyth_over_wormhole.md:50]:
```suggestion
| Solitaire                 | A convenient Rust framework for contract development that greatly simplifies Solana's unique execution model; A by-product of Wormhole's Solana efforts, built on top of the official Solana APIs. | https://github.com/certusone/wormhole |
| Wormhole's Solana program | The Solana end of Wormhole, built with Solitaire.                                                                                                                                                  | https://github.com/certusone/wormhole |
```

File: whitepapers/0007_pyth_over_wormhole.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+single, aggregated number from the upper-level asset price
+description.
+
+In the current system metadata from arbitrary Pyth feeds can not be
+trusted and anyone can create new feeds for popular symbols. Projects
+that build products on top of Pyth need to maintain a list of the
+product_ids (Solana account keys) that they trust.  As long as this is
+the case we can also rely on projects to hardcode the relevant
+metadata that they require in their business logic and focus this
+initial release on just attesting pricefeeds.
```
Comment (SEJeff) [inline whitepapers/0007_pyth_over_wormhole.md:77]: Is this the case? I thought the pyth foundation or someone had to create new product metadata and permission it.

File: whitepapers/0007_pyth_over_wormhole.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+#### High level user API
+* `attest(product_id, price_id)` - Attest the metadata of the
+  specified price account.
+* `last_attestation(product_id, price_id)` - Look up the last attested
+  price data for a price account (if present); Free of charge
+  off-chain.
+
+#### Detailed Code level documentation
+The Rust code documentation for pyth2wormhole can be found
+[here (TBD, ideally rust's cargo-doc UI for pyth2wormhole crates)](https://example.com).
```
Comment (SEJeff) [inline whitepapers/0007_pyth_over_wormhole.md:186]: Ummmmmmm

**Review by SEJeff — APPROVED** [review]
Small nits, but worth fixing.

File: whitepapers/0007_pyth_over_wormhole.md (hunk 2 above)
Comment (drozdziak1) [reply whitepapers/0007_pyth_over_wormhole.md:77]: Kinda, def need to reword this.

**Comment by ali-bahjati** [comment]
@drozdziak1 Can you update it to also include Batch Price Attestation?

**Comment by evan-gray** [comment]
Closing, as Pyth2Wormhole is moving to a separate repo 🙂

**Pull request closed without merging**